packages/cwp-template-aws/template/common/apps/admin/src/App.scss

```diff
@@ -13,7 +13,7 @@ $webiny-theme-light-text-primary-on-background: rgba(0, 0, 0, 0.87);
 $webiny-theme-light-text-secondary-on-background: rgba(0, 0, 0, 0.54);
 $webiny-theme-light-text-hint-on-dark: rgba(255, 255, 255, 0.5);
 $webiny-theme-light-caret-down: url("data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%23000000%22%20fill-rule%3D%22evenodd%22%20opacity%3D%220.54%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E");
-$webiny-theme-typography-font-family: "Source Sans Pro";
+//$webiny-theme-typography-font-family: "Source Sans Pro";
```

Comment:
Hm... why this change?

I guess we'll get back to this once we start wrapping things up?

Reply:
While working on this PR, I've noticed that the new `Inter` font family is often replaced by `Source Sans Pro`. At the moment, I have decided to comment it out. As you suggested, I added reminder in GH project